ddsp/core.py

```diff
@@ -1488,4 +1494,4 @@ def sinc_filter(audio: tf.Tensor,
                                            window_size=window_size,
                                            sample_rate=sample_rate,
                                            high_pass=high_pass)
-  return fft_convolve(audio, impulse_response, padding=padding)
+  return fft_convolve(audio, impulse_response, padding=padding)
```

Comment:
not sure what's changed here, maybe revert it?

Reply:
This is because I accidentally deleted a new line. Added a newline to the end of core.py.